docs/integration/KafkaIntegration.md

```diff
@@ -8,104 +8,164 @@ sidebar_position: 1
+  It means that it looks for schemas available at `<topic-name>-(key or value)` *subject*. For example for topic `transactions`, it looks for schemas at `transactions-key` *subject* for key and `transactions-value` *subject* for value
+- In the Azure Schema Registry, *subject* concept doesn't exist - schemas are grouped by the same *schema name*. Because of that, Nussknacker introduces
+  own convention for association between schema and topic: *schema name* should be in format: `CamelCasedTopicNameKey` for keys and `CamelCasedTopicNameValue` for values.
+  For example for `input-events` topic, schema name should be named `InputEventsKey` for key or `InputEventsValue` for value. Be aware that it may require change of schema name
+  not only in Azure portal but also inside schema content - those names should be the same to make serialization works correctly
+
+## Connection and Authentication Configuration
 
-### Connection configuration
+Under the hood Nussknacker uses `kafkaProperties` to configure standard kafka client. It means that all standard Kafka client properties will be respected.
+For detailed instruction where it should be placed inside Nussknacker's configuration, take a look at [Configuration details section](#configuration-details)
 
-Configuration necessary to connect with Schema registry should be placed inside `kafkaProperties`. For detailed instruction
-how configuration is handled take a look at [Configuration paragraph](#configuration)
+### Kafka - Connection
 
-#### Confluent-based implementation
+To configure connection to kafka, you need to configure at least `bootstrap.servers` property. It should contain comma separated list of urls to Kafka brokers.
+
+#### Kafka - Authentication
+
+Kafka cluster has multiple options to configure Authentication. Take a look at [Kafka security documentation](https://kafka.apache.org/090/documentation.html#security)
+to see detailed examples how those options should be translated into properties. For example for the typical `SASL_SSL` configuration with
+credential in `JAAS` format, you should provide configuration similar to this one:
+
+```
+kafkaProperties {
+  "schema.registry.url": "http://schemaregistry:8081"
+  "bootstrap.servers": "broker1:9092,broker2:9092"
+  "security.protocol": "SASL_SSL"
+  "sasl.mechanism": "PLAIN"
+  "sasl.jaas.config": "org.apache.kafka.common.security.plain.PlainLoginModule required username=\"some_user\" password=\"some_user\";"
+}
+```
+
+If you use Azure Events Hubs (which uses this mode), username will be `$ConnectionString` and password will be the connection string starting with `Endpoint=sb://`.
+
+In case if you use your own CA and client+server certificates authentication, you should additionally provide:
+`ssl.keystore.location`, `ssl.keystore.password`, `ssl.key.password`, `ssl.truststore.location`, `ssl.truststore.password`.
+
+To make sure if your configuration is correct, you can test it with standard kafka-cli commands
```

Comment:
make sure if -> check if
or
make sure if -> make sure that

Reply:
This step is not mandatory, because of that I've added "To". Should it be e.g "To make sure  that your ..." or should I change it in other way?